public/app/features/dashboard/components/PanelEditor/PanelOptionsEditor.tsx

```diff
@@ -57,11 +57,12 @@ export const PanelOptionsEditor: React.FC<PanelOptionsEditorProps<any>> = ({
               return null;
             }
 
-            const label = (
-              <Label description={e.description} category={e.category?.slice(1)}>
-                {e.name}
-              </Label>
-            );
+            const label =
```

Comment:
This change will only show the label if name|description exists.  This allows me to make the `<Label ` explicit in the body depending on the option state